src/services/storage/storage-migration.ts

```diff
@@ -1,3 +1,11 @@
-export default function migrate(): Promise<void> {
-  return Promise.resolve()
+import { AccountDapps } from '../../model/storage/dapps.model'
+import { Version } from '../../model/storage/version.model'
+import { accountDappsFactory } from './storage-factories'
+
+export function migrateDapps(accountDapps: AccountDapps, { major, minor }: Version): AccountDapps {
```

Comment:
I intentionally left that, because for now we don't need it. But in the future it will be required for sure. And additionally, to keep the same structure of all `migration` functions.